src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -557,9 +562,26 @@ func (a *StorageMinerActorCode_I) _verifySeal(rt Runtime, onChainInfo sector.OnC
 		WindowCount_: info.WindowCount(),
 		Partitions_:  info.SealPartitions(),
 	}
+
+	// @param address addr.Address info.Worker()
+	getActorIDParams := make([]util.Serialization, 1)
+	panic("TODO: serialize arguments")
+
+	getActorIDReceipt := rt.SendPropagatingErrors(&vmr.InvocInput_I{
+		To_:     addr.InitActorAddr,
+		Method_: sys.Method_InitActor_GetActorIDForAddress,
+		Params_: getActorIDParams,
+	})
+
+	getActorIDRet := getActorIDReceipt.ReturnValue()
+	minerID, err := addr.Deserialize_Address(getActorIDRet)
+	if err != nil {
+		rt.AbortStateMsg("sm.verifySeal: failed to get actor id")
+	}
+
 	svInfo := sector.SealVerifyInfo_I{
 		SectorID_: &sector.SectorID_I{
-			MinerID_: info.Worker(), // TODO: This is actually miner address. MinerID needs to be derived.
+			MinerID_: minerID,
```

Comment:
are you sure we want the minerID, not workerinfo?

We should homogeonize throughout the spec whether we call it `minerID` or `minerAddr` both are used and mean same. Based on your choice, please change accordingly in the `keyStore` file

Reply:
will leave this to @porcuquine